Description: Upstream changes introduced in version 1.4.1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 caml2html (1.4.1-1) unstable; urgency=low
 .
   * Initial release. (Closes: #605741)
 .
 The person named in the Author field signed this changelog entry.
Author: Sylvain Le Gall <gildor@debian.org>
Bug-Debian: http://bugs.debian.org/605741

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- caml2html-1.4.1.orig/Makefile
+++ caml2html-1.4.1/Makefile
@@ -57,24 +57,25 @@ version.ml: version.ml.mlx Makefile
 
 byte: caml2html.byte
 
+CAML2HTML=$(word 1,$(wildcard ./caml2html ./caml2html.byte) false)
 test:
 	rm -f caml2html_test.mli
 	ocamlc -i caml2html_test.ml > caml2html_test.mli
 	ocamlc -c caml2html_test.mli
 	ocamlc -c -dtypes caml2html_test.ml
-	./caml2html -o caml2html_test.html \
+	$(CAML2HTML) -o caml2html_test.html \
 		caml2html_test.mli caml2html_test.ml caml2html_test2.ml \
 		-ln -ie7 \
 		-ext date:date \
 		-ext cat:cat \
 		-ext "rot13:tr '[a-z]' '[n-za-m]'"
-	./caml2html -o caml2html_self_test.html \
+	$(CAML2HTML) -o caml2html_self_test.html \
 		tag.ml annot.mli annot.ml plugin.mli plugin.ml \
 		input.mli input.mll output.mli output.ml \
 		output_latex.mli output_latex.ml \
 		main.ml \
 		-ln
-	./caml2html -latex -o caml2html_self_test.tex \
+	$(CAML2HTML) -latex -o caml2html_self_test.tex \
 		tag.ml annot.mli annot.ml plugin.mli plugin.ml \
 		input.mli input.mll output.mli output.ml \
 		output_latex.mli output_latex.ml \
@@ -85,7 +86,7 @@ caml2html: $(OBJS-NAT)
 	$(OCAMLOPT) -o caml2html str.cmxa unix.cmxa $(OBJS-NAT)
 
 caml2html.byte: $(OBJS)
-	$(OCAMLC) -custom -o caml2html.byte str.cma unix.cma $(OBJS)
+	$(OCAMLC) -o caml2html.byte str.cma unix.cma $(OBJS)
 
 lib: all bytelib optlib
 
@@ -144,7 +145,7 @@ input.ml: input.mll
 P = caml2html-$(VERSION)
 
 caml2html.html: caml2html caml2html.html.mlx
-	./caml2html -help > caml2html-help
+	$(CAML2HTML) -help > caml2html-help
 	camlmix -o caml2html.html caml2html.html.mlx
 
 archive: pre opt test caml2html.html
--- caml2html-1.4.1.orig/caml2html.mli
+++ caml2html-1.4.1/caml2html.mli
@@ -176,10 +176,10 @@ val ocaml :
   Buffer.t -> 
   Input.token list -> unit
 (** [ocaml buf l] formats the list of tokens [l] into some HTML code
-  which should be placed in a <code> or <pre> region,
+  which should be placed in a <code\> or <pre\> region,
   and adds the result the given buffer [buf].
   Option [nbsp] tells if the spaces must be converted into "&nbsp;" or not
-  (required in <code> regions but not in <pre>; default is false). *)
+  (required in <code\> regions but not in <pre\>; default is false).  *)
 
 val ocamlcode :
   ?annot:Annot.tag list ->
--- caml2html-1.4.1.orig/output.mli
+++ caml2html-1.4.1/output.mli
@@ -69,24 +69,24 @@ val ocaml :
   Buffer.t -> 
   Input.token list -> unit
 (** [ocaml buf l] formats the list of tokens [l] into some HTML code
-  which should be placed in a <code> or <pre> region,
+  which should be placed in a <code\> or <pre\> region,
   and adds the result the given buffer [buf].
   Option [nbsp] tells if the spaces must be converted into "&nbsp;" or not
-  (required in <code> regions but not in <pre>; default is false). *)
+  (required in <code\> regions but not in <pre\>; default is false). *)
 
 val ocamlcode :
   ?annot:Annot.tag list ->
   ?keyword_colors:(string, string option * string option * string) Hashtbl.t ->
   ?param:param -> ?tag_open:string -> ?tag_close:string -> string -> string
 (** [ocamlcode s1 s2] parses [s1] and formats the result as a HTML string
-  enclosed between <code> and </code> unless specified otherwise. *)
+  enclosed between <code\> and </code\> unless specified otherwise. *)
 
 val ocamlpre :
   ?annot:Annot.tag list ->
   ?keyword_colors:(string, string option * string option * string) Hashtbl.t ->
   ?param:param -> ?tag_open:string -> ?tag_close:string -> string -> string
 (** [ocamlcode s1 s2] parses [s1] and formats the result as a HTML string
-  enclosed between <pre> and </pre> unless specified otherwise. *)
+  enclosed between <pre\> and </pre\> unless specified otherwise. *)
 
 (* $Id: output.mli 12 2010-06-28 18:18:41Z mjambon $ *)
 
